<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>461</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Petri Net Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="separator"/>
    <addaction name="actionOpenLocally"/>
    <addaction name="actionSaveLocally"/>
    <addaction name="separator"/>
    <addaction name="actionOpenRemote"/>
    <addaction name="actionSaveRemote"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuNastaven">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionNetInformation"/>
    <addaction name="actionConnectToServer"/>
    <addaction name="actionDisconnectFromServer"/>
    <addaction name="actionLogin"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuSoubor"/>
   <addaction name="menuNastaven"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMouse"/>
   <addaction name="actionPlace"/>
   <addaction name="actionTransition"/>
   <addaction name="actionArc"/>
   <addaction name="separator"/>
   <addaction name="actionSimulate"/>
   <addaction name="actionSimulate_Step"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenLocally">
   <property name="text">
    <string>&amp;Open locally</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveLocally">
   <property name="text">
    <string>&amp;Save locally</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenRemote">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open remote</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionSaveRemote">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save remote</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionMouse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mouse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionPlace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Place</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionTransition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionArc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arc</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;Nov√Ω</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="text">
    <string>&amp;New tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>&amp;Back</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionNetInformation">
   <property name="text">
    <string>Net &amp;information</string>
   </property>
  </action>
  <action name="actionConnectToServer">
   <property name="text">
    <string>&amp;Connect to server</string>
   </property>
  </action>
  <action name="actionDisconnectFromServer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disconnect from server</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </action>
  <action name="actionSimulate">
   <property name="text">
    <string>Simulate</string>
   </property>
  </action>
  <action name="actionSimulate_Step">
   <property name="text">
    <string>Simulate Step</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
